ERROR - 2025-06-29 10:40:30 --> mysqli_sql_exception: Unknown column 'user.id' in 'where clause' in /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/BaseModel.php(613): CodeIgniter\Model->doFind(true, '1')
#6 /opt/lampp/htdocs/LBS/app/Controllers/Api/Auth.php(122): CodeIgniter\BaseModel->find('1')
#7 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Api\Auth->get_user()
#8 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Auth))
#9 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 /opt/lampp/htdocs/LBS/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /opt/lampp/htdocs/LBS/public/index.php(62): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-06-29 10:40:30 --> Error: Call to a member function getFirstRow() on bool
[Method: POST, Route: api/auth/get_user]
in SYSTEMPATH/Model.php on line 211.
 1 SYSTEMPATH/BaseModel.php(613): CodeIgniter\Model->doFind(true, '1')
 2 APPPATH/Controllers/Api/Auth.php(122): CodeIgniter\BaseModel->find('1')
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Api\Auth->get_user()
 4 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Auth))
 5 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(62): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 12:32:13 --> ErrorException: Attempt to read property "user_psw" on array
[Method: POST, Route: api/auth/login]
in APPPATH/Controllers/Api/Auth.php on line 65.
 1 APPPATH/Controllers/Api/Auth.php(65): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "user_psw" on array', '/opt/lampp/htdocs/LBS/app/Controllers/Api/Auth.php', 65)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Api\Auth->login()
 3 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Auth))
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(62): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 12:33:05 --> ErrorException: Attempt to read property "user_id" on array
[Method: POST, Route: api/auth/login]
in APPPATH/Controllers/Api/Auth.php on line 66.
 1 APPPATH/Controllers/Api/Auth.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "user_id" on array', '/opt/lampp/htdocs/LBS/app/Controllers/Api/Auth.php', 66)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Api\Auth->login()
 3 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Auth))
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(62): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 12:33:55 --> ErrorException: Attempt to read property "user_id" on array
[Method: POST, Route: api/auth/login]
in APPPATH/Controllers/Api/Auth.php on line 78.
 1 APPPATH/Controllers/Api/Auth.php(78): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "user_id" on array', '/opt/lampp/htdocs/LBS/app/Controllers/Api/Auth.php', 78)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Api\Auth->login()
 3 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Auth))
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(62): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
